.gitignore

```diff
@@ -1,9 +1,33 @@
+# phpstorm project files
+.idea
+
+# netbeans project files
+nbproject
+
+# zend studio for eclipse project files
+.buildpath
+.project
+.settings
+
+# windows thumbnail cache
+Thumbs.db
+
```

Comment:
imo stuff like this should go into a global gitignore file of user config, not in each repo.